docs/UserGuide.md

```diff
@@ -131,29 +127,127 @@ The table below shows a list of command parameters that will be used in this fea
 |`SERVICE_CODE`| The service code is the code that identifies a particular type of service provided. <br> <br> It must be alphanumeric words of 5 characters long. | E.g. If you have added an eyelash extension service into GrAB3 and its service code is `SC001`. <br> <br> Typing `SC001` would refers to the eyelash extension service.
 |`ORDER` | The order refers to ascending or descending. <br> <br> It must be in the format of `asc` or `desc` | E.g. Typing `asc` would mean ascending.  <br> <br> E.g. Typing `desc` would mean descending.
 
-##### Find Revenue : `findrev`
+##### Find a Revenue : `findrev`
 
-You can use this command to find revenue by date or service code.
+You can use this command to find revenues by 'date' or 'service code' in GrAB3.
 
 Format : `findrev [dt/DATE]* [s/SERVICE_CODE]*`
 
-|<div markdown="block" class="alert alert-info"> :information_source:</div> | Refer to [Revenue Tracker Command Parameters](#revenue-tracker-command-parameters) for more details about each parameter.
-|---------|---------
-<br> 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+* Refer to [Revenue Tracker Command Parameters](#revenue-tracker-command-parameters) for more details about each parameter.
+
+</div>
+
 Example :
 
 You have just stopped operations for the day, and you wish to view all the revenues generated for today (29-09-2020).
 
 You can follow the steps below to get the list of revenues for the day.
 
 Steps :
-1. Type `findrev dt/29-09-2020` into the command box.
+1. Type `findrev dt/29-09-2020` into the _Command Box_.
 1. Press `Enter` to execute. 
 
 Outcome : 
-1. It will display a success message ~~in the *Dashboard*.~~
-1. GrAB3 will list out all the revenue for 29-09-2020 ~~in the *Revenue* tab.~~  
-1. You will be able to see all the revenue for 29-09-2020.
+1. It will display a success message ~~in the _Dashboard_.~~
+1. GrAB3 will list out all the revenue for 29-09-2020 in the _Revenue_ tab.  
+
+{Example outcome screenshot}
+
+##### Sort Revenues: `sortrev`
```

Comment:
See #62 from Hans, he included a numbering scheme for the features like 4.1.1, 4.1.2, etc. I think you can edit this section after his PR is merged into master.

Reply:
I will do this in the last milestone, issue created